src/libsync/networkjobs.cpp

```diff
@@ -713,7 +713,7 @@ SimpleNetworkJob::SimpleNetworkJob(AccountPtr account, const QUrl &rootUrl, cons
 SimpleNetworkJob::SimpleNetworkJob(AccountPtr account, const QUrl &rootUrl, const QString &path, const QByteArray &verb, QByteArray &&requestBody, const QNetworkRequest &req, QObject *parent)
     : SimpleNetworkJob(account, rootUrl, path, verb, new QBuffer(&_body), req, parent)
 {
-    _body = requestBody;
+    _body = std::move(requestBody);
```

Comment:
This cast seems redundant, since `requestBody` is already an rvalue reference, thus causing `QByteArray::operator=(QByteArray&&)` (move-assignment operator) to be called on `_body`. I think this is to satisfy SonarCloud only, so maybe we should put a comment above it.

Reply:
We can either disable sonar cloud or follow its advice in 1 of 10000 cases.
https://sonarcloud.io/project/issues?issues=AX8MOGEA1OuwF-0ACQyA&open=AX8MOGEA1OuwF-0ACQyA&id=owncloud_client

Reply:
Hm, I don't see how it's technically going to change anything, but the C++ core guidelines recommend this change. Fascinating.